test/affinity.cxx

```diff
@@ -170,11 +170,11 @@ TEST_CASE("pinning", "[affinity_t]")
 		REQUIRE(result == static_cast<int32_t>(processor));
 	}
 #elif defined(_WIN32)
-	std::size_t count{0};
+	uint32_t count{0};
 	for (const auto &processor : *affinity)
 	{
 		const auto result = std::async(std::launch::async,
-			[&](const uint32_t currentProcessor) noexcept -> GROUP_AFFINITY
+			[&](uint32_t currentProcessor) noexcept -> GROUP_AFFINITY
```

Comment:
Is there a particular reason for removing this `const`?

Reply:
That surprises us as uint32_t being trivial should mean there's no penalty, but if it actually hurts things that badly, then we'll allow it despite it stepping down the lambda's safety a touch